script/run-comp-match

```diff
@@ -16,20 +17,85 @@ sys.path.insert(1, str(REPO_ROOT / 'modules'))
 import controller_utils  # noqa:E402 # isort:skip
 from controller_utils import NUM_ZONES  # isort:skip
 
+NUM_CONTROLLERS = NUM_ZONES + 2
```

Comment:
Is there a purpose for this line as the constant appears to be unused.

Reply:
Nope. 11dc5e6